tests/openai/test_openai_model_export.py

```diff
@@ -225,7 +333,40 @@ def test_no_variables(tmp_path):
     assert list(map(json.loads, model.predict(data))) == expected_output
 
 
-def test_no_messages(tmp_path):
+def test_prompt_no_variable(tmp_path):
+    mlflow.openai.save_model(
+        model="text-davinci-003",
+        task=openai.Completion,
+        path=tmp_path,
+        suffix="Say ",
```

Comment:
Is this included to demonstrate that kwargs will accept a random argument passed?

Reply:
This is a great catch! `suffix` is a valid argument for task `completion` but in the mock OpenAI API we implemented we don't have support for it. I will remove it from the test for now.